plugins/route-prefix-context-plugin.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+
+const envVarsPlugin = require('./environment-variables-plugin');
+
+module.exports = function() {
+  const envVars = envVarsPlugin().of();
+  return function middleware(ctx, next) {
```

Comment:
Goal is to strip the prefix from `ctx.url` and supply the prefix in `ctx.prefix` for any consumers that might rely on the prefix.